lib/clone.js

```diff
@@ -49,21 +52,24 @@ const cloneTask = (sshGitHubUrl, repoName, progressBar) => {
 const clone = (services, task = cloneTask) => {
   process.stdout.write(`Cloning repos...${newline}`)
 
-  const gitHubDetails = getGitHubDetails(services)
+  const gitHubRepoInfo = getGitHubDetails(services)
   const progressBar = new Progress(':percent [:bar] (:current/:total)', {
-    total: gitHubDetails.length,
+    total: gitHubRepoInfo.length,
     clear: true
   })
+  const githubInfoParentArray = chunkArray(gitHubRepoInfo, numberOfCPUs * 4)
```

Comment:
My main concern was that this thread is the only place where someone could know why it's `* 4`. Why not `* 16` for example?

Yeah I was going to raise an issue anyway